spec/index.bs

```diff
@@ -27,6 +27,14 @@ Introduction {#intro}
 
 This specification rocks.
 
+Convention {#convention}
+========================
+
+## Coordinate Systems ## {#coordinate-systems}
+WebGPU's coordinate systems match DirectX and Metal's coordinate systems in graphics pipeline.
+  - Y-axis is up in normalized device coordinate (NDC): point(-1.0, -1.0) in NDC is located at the bottom-left corner.
+    In addition, x and y in NDC must be between -1.0 and 1.0 inclusive, while z in NDC must be between 0.0 and 1.0 inclusive.
+  - Y-axis is down in all other coordinate systems: origin(0, 0) is located at the top-left corner in these coordinate systems.
```

Comment:
It would be good to be more concrete about "all other" here. It's a specification, it should leave space for guesswork.
So NDC is specified here. Framebuffer coordinates should be specified here. Window/present coordinate - for now we should just say it matches the framebuffer coordinates.
Texture coordinates - there is no up/down there. We should just say that UV of (0, 0) corresponds to the texel that is first in texture memory.